homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -255,6 +257,12 @@ def _async_register_button_devices(
     return button_devices_by_dr_id, device_info_by_device_id
 
 
+def _handle_keypad_serial(keypad_device: dict, bridge_serial: int) -> str:
```

Comment:
Can you pick a name here that better shows the intent ?

Reply:
updated to _handle_none_keypad_serial